pages/index.js

```diff
@@ -244,7 +244,8 @@ const Home = (props) => {
                 <MailIcon className={styles.icon} />
                 <p>nusfintech.ops@gmail.com</p>
               </div>
-              <form onSubmit={handleSubmit(onSubmit)} autoComplete="off">
+              
+              {/* <form onSubmit={handleSubmit(onSubmit)} autoComplete="off">
```

Comment:
Since onSubmit() function is no longer used, let's remove it from this file. Also could you search for all occurences of **import request from "util/request"** in the repo and remove them too, since we are no longer using API calls.